frontend/src/routes/AppRoutes.tsx

```diff
@@ -9,17 +9,32 @@ import { Dashboard, Login, NotFound, Profile } from '../pages';
 import { Loading } from '@/components';
 import { PublicRoute } from './PublicRoute';
 import { PrivateRoute } from './PrivateRoute';
+import { getKeycloakInfo } from '@/services';
 
 export default () => {
-  return (
-    <ReactKeycloakProvider
-      authClient={
-        new Keycloak({
-          realm: import.meta.env.VITE_KEYCLOAK_REALM,
-          url: import.meta.env.VITE_KEYCLOAK_AUTH_URL,
-          clientId: import.meta.env.VITE_KEYCLOAK_CLIENT,
-        })
+  const [keycloakInfo, setKeycloakInfo] = useState<Keycloak>();
```

Comment:
This looks good. My only concern on this approach is having some sort of fallback page or handling the error from a UI perspective in case the request fails?

Reply:
(not necessarily needed on this PR for approval, just something to consider in general I suppose)

Reply:
Oh sorry I see that you have a fallback message. Maybe we can get Chevonne to design error/not found pages? Or maybe there is a fallback gov one we can borrow (but again - not for this PR)

Reply:
Yes will need to look into options! An error page might be okay.